components/common/EventPopup/AcmEventHeads.tsx

```diff
@@ -87,55 +86,34 @@ export default function AcmEventHeads({teamTitle, pills, setPills, setEmptyError
     }
 
     return (
-        <div style={{
-            display: "flex",
-            margin: "30px",
-        }}>
-            <div style={{
-                width: "300px"
-            }}>
-                <div style={{
-                    width:"300px",
-                    padding:"20px"
-                }}>
-                    <CustomTextField
-                        type="text"
-                        onChange={ handleNameSearchChange }
-                        value={nameSearchValue}
-                        label={`Search ${teamTitle}`}
-                        fullWidth={true}
-                        variant="filled"
-                    />
+        <div className={styles['acm-event-heads-container']}>
+            <div className={styles.pillInputDiv}>
+                <PillContainer handleDelete={handleDelete} pills={pills} />
+                <input
+                    type="text"
+                    onChange={handleNameSearchChange}
+                    value={nameSearchValue}
+                    className={styles.pillInput}
+                    placeholder={`Search ${teamTitle}*`}
+                />
+            </div>
 
-                    <div>
-                        {
-                            searchResult.map((pill: Pill) =>
-                                <div
-                                    onClick={() => addUserToPills(pill)}
-                                    key={pill.email}
-                                    style={{
-                                        height: "40px",
-                                        borderRadius: "10px",
-                                        padding: "5px"
-                                    }}
-                                >
-                                    {pill.name}
-                                    <p
-                                        style={{
-                                            color:"grey",
-                                            fontSize:"small",
-                                        }}
-                                    >
-                                        {pill.email}
-                                    </p>
-                                </div>)
-                        }
-                    </div>
+            <div>
+                <div className={styles['pills-container']}>
```

Comment:
the search results should hide away if the focus changes from Search Organizing/Publicity Heads text box